config/site.php

```diff
@@ -8,36 +8,21 @@
         'menu' => [
             'items' => [
                 'community' => [
-                    'team' => [
+                    'getInvolved' => [
                         'url' => [
```

Comment:
This should also be a static URL, sorry I didn't notice earlier.

Reply:
Done
Changed all links